packages/fx-core/src/plugins/resource/bot/plugin.ts

```diff
@@ -135,18 +135,9 @@ export class TeamsBotImpl implements PluginImpl {
     );
     await handler?.start(ProgressBarConstants.PROVISION_STEP_START);
 
-    // 0. Check Resource Provider
-    const azureCredential = await this.getAzureAccountCredenial();
```

Comment:
Just confirm if there is any method not used by other code (e.g. createResourceProviderClient(), ensureResourceProvider()), we can safely remove it.

Reply:
bot/index.js has used this method